src/main/java/com/ita/if103java/ims/config/GlobalExceptionHandler.java

```diff
@@ -29,4 +29,11 @@ public ResponseEntity handleCRUDException(HttpServletRequest req, Exception e) {
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
             .body(new String[]{e.getMessage()});
     }
+
+    @ExceptionHandler({UserOrPasswordIncorrectException.class})
+    public ResponseEntity handleUserOrPasswordIncorrectException(HttpServletRequest req, Exception e) {
+        LOGGER.warn(e.getMessage(), e);
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED)
+            .body(new String[]{e.getMessage()});
```

Comment:
In the case we have:
```java
new String[]{e.getMessage()}
```
our json looks like:
```json
[
  "Error during a ...."
]
```
Suggestion is to add new model
```java
class ErrorMessageDto {
      private String message;
      ...
}
```
to have the next output:
```json
{ 
    "message": "Error during a ...."
}
```

Reply:
I wanted to do it in the near future, but I consider this Pull request a quick fixing.